resources/site-packages/quasar/daemon.py

```diff
@@ -181,6 +181,16 @@ def start_quasard(**kwargs):
         kwargs["env"] = env
         kwargs["close_fds"] = True
 
+    wait_counter = 0
+    while xbmc.getCondVisibility('Window.IsVisible(10140)'):
+        log.debug('Addon Settings opened, waiting before start')
```

Comment:
Over-capitalization ;) Should be `Add-on settings opened, waiting before starting`